SECURITY_CONTACTS

```diff
@@ -14,3 +14,4 @@ brancz
 logicalhan
 ehashman
 dashpole
+rexagod
```

Comment:
could you also update the list of security contacts to match the current approvers?